card/src/main/java/com/adyen/checkout/card/internal/ui/view/CardView.kt

```diff
@@ -405,35 +403,14 @@ class CardView @JvmOverloads constructor(
 
     private fun initCardBrandLogoViews(selectedIndex: Int) {
```

Comment:
Looking at the designs, shouldn't we remove this logic to show the selected brand inside of the input field?

Reply:
There was some confusion due to ticket having old designs, fixed now.